pkg/client/imagestreams.go

```diff
@@ -139,7 +140,7 @@ func transformUnsupported(err error) error {
 	// enabled by policy. A create request will return a Forbidden(403) error.
 	// We want to return ErrImageStreamImportUnsupported to allow fallback behavior
 	// in clients.
-	if apierrs.IsForbidden(err) {
```

Comment:
Having to change this means that clients from 1.1.1 -> 1.1.6 are broken if this change goes in.